### PR TITLE
HOTFIX: Checkstye fixes follow up for KAKFA-2531.

### PR DESCRIPTION
**Pull request by ewencp**

### CODE REVIEW

**Comment by ewencp** [comment]
@gwenshap Some minor checkstyle issues were missed in KAFKA-2531, probably because of the other transient test failures that kept checkstyle from running in the Jenkins tests.

File: checkstyle/import-control.xml [hunk 1]
```diff
@@ -108,6 +108,7 @@
 			<allow pkg="org.apache.kafka.clients.consumer" />
             <allow pkg="com.fasterxml.jackson" />
             <allow pkg="net.sourceforge.argparse4j" />
```
Comment (ijuma) [inline checkstyle/import-control.xml:110]: Is it worth fixing these bad indents at the same time?

**Comment by gwenshap** [comment]
Completely agree that we need to fix the bad indents, but merging this PR anyway to unblock the build. 

@ijuma feel free to throw in a PR for the indents.

**Comment by ijuma** [comment]
@gwenshap There you go:

https://github.com/apache/kafka/pull/253

**Pull request closed without merging**